lib/partition_setup.pm

```diff
@@ -190,8 +190,13 @@ sub addlv {
 # naturally sees it as a "disk". We have to uncheck the "CDROM".
 sub unselect_xen_pv_cdrom {
     if (check_var('VIRSH_VMM_TYPE', 'linux')) {
-        assert_screen 'select-hard-disk';
```

Comment:
Why not to match this tag? Just to verify that dialog is opened?

Reply:
OK, PR updated & revalidated. We need following needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/728